apps/jobs/views.py

```diff
@@ -280,6 +280,9 @@ def change_submission_data_and_visibility(
                 "error": "Sorry, cannot accept submissions since challenge phase is not public"
             }
             return Response(response_data, status=status.HTTP_403_FORBIDDEN)
+        elif 'baseline_submission' in request.data:
+            response_data = {'error': 'Sorry, you are not authorized to make this request!'}
```

Comment:
Please change `'error': 'Sorry, you are not authorized to make this request!` to `'error': 'Sorry, you are not authorized to make this request`